src/modules/inputs/Input.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './Input.css';
+
+const Input = (props) => {
+    const { type, input } = props;
+
+    return (
+        <input
+            className="j-input"
```

Comment:
There is nothing except the single `<input>` element. I don't think that extra 2 lines(import `bem-cn` and declaring the `block`) will make this code better and more readable.